drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -1612,7 +1619,10 @@ int usb_dc_ep_enable(const uint8_t ep)
 		return -EINVAL;
 	}
 
-	nrfx_usbd_ep_dtoggle_clear(ep_addr_to_nrfx(ep));
+	if (!NRF_USBD_EPISO_CHECK(ep)) {
+		/* ISO endpoints do not support dtoggle */
```

Comment:
The comment does not explain it correct, see "8.5.5 Isochronous Transactions". Probably: "A full-speed device do not support toggle sequencing and should only send DATA0 PID.

Reply:
That's right for both comments. Will update it, thanks!

Reply:
What about?
```
/* ISO transactions for full-speed device do not support
 * toggle sequencing and should only send DATA0 PID.
 * There is no point to clear it.
 */
```

Reply:
/* ISO transactions for full-speed device do not support
 * toggle sequencing and should only send DATA0 PID.
 */